tests/unit_tests/ensemble_evaluator/test_ensemble_evaluator_config.py

```diff
@@ -55,6 +55,13 @@ def test_that_get_machine_name_is_predictive(mocker):
     # as an implementation detail.
     expected_resolved_name = ptr_records[0].rstrip(".")
 
+    # Avoid possibility of flakyness in code paths not relevant
+    # for this test:
+    mocker.patch("socket.gethostname", return_value=None)
```

Comment:
Down the line will this will trigger a general exception? At least we looking at this: `/python3.8/site-packages/dns/reversename.py`

Reply:
Since we mock that one as well, I don't' think an exception can occur.

Reply:
Right, I missed that one :)